src/schema/v2/artwork/utilities.ts

```diff
@@ -78,12 +78,16 @@ export const getFigures = ({
   images,
   external_video_id,
   set_video_as_cover,
+  includeAll = false,
```

Comment:
It's been already added for `images`. For the artwork detail page we also need `figures` to return all images (including processing images).